join-welcome/handler.py

```diff
@@ -6,71 +6,79 @@
 import hmac
 from time import perf_counter
 
-import requests
-
-def handle(req):
-    """handle a request to the function
-    Args:
-        req (str): request body
-    """
-
-    log_event(req)
+def handle(event, context):
+    # TODO implement
+    log_event(event.body)
 
     if os.getenv("log_env", "0") == "1":
         log_env()
 
     r = None
     try:
-        r = json.loads(req)
+        r  = json.loads(event.body)
     except ValueError:
         sys.stderr.write("Error parsing request, invalid JSON")
-        return "Error parsing request, invalid JSON"
```

Comment:
Give it a status code too.. 400?

Reply:
I thought I did return a 400? This is for the removed code

Reply:
That looks like returning a string?

Reply:
This is what is being returned no?
```python
try:
        r = json.loads(event.body)
except ValueError:
        sys.stderr.write("Error parsing request, invalid JSON")
        return {
            "statusCode": 400,
            "body": "Error parsing request, invalid JSON"
        }
```

Reply:
My bad. I misread it.